.travis.yml

```diff
@@ -15,7 +15,7 @@ matrix:
   fast_finish: true
 
 install:
-  - pip install pylint
+  - pip install -r requirements-dev.txt
```

Comment:
Maybe try `black>=18.0` instead? Also I think the `-dev` requirements should be the bottom one, since at least in principle, the `-dev` requirements are what you install afterwords if you decide you are going to also develop futaba.